src/pages/learn/serving-over-http.mdx

```diff
@@ -34,37 +77,63 @@ http://myapi/graphql?query={me{name}}
 
 Query variables can be sent as a JSON-encoded string in an additional query parameter called `variables`. If the query contains several named operations, an `operationName` query parameter can be used to control which one should be executed.
 
-### `POST` request
+#### Choosing an HTTP method
 
-A standard GraphQL POST request should use the `application/json` content type, and include a JSON-encoded body of the following form:
+When choosing an HTTP method for a GraphQL request, there are a few points to consider. First, support for HTTP methods other than `POST` will be at the discretion of the GraphQL server, so a client will be limited to the supported verbs.
 
-```json
-{
-  "query": "...",
-  "operationName": "...",
-  "variables": { "myVariable": "someValue", ... }
-}
-```
+Additionally, the `GET` HTTP method may only be used for query operations, so if a client sends a mutation operation in a document then it must use the `POST` method instead.
```

Comment:
Subtly incorrect, the document may contain mutations and be sent over GET so long as the chosen operation to execute (via `operationName`) is a query operation.

```suggestion
Additionally, the `GET` HTTP method may only be used for `query` operations, so if a client requests execution of a `mutation` operation then it must use the `POST` method instead.
```